### PR TITLE
gacl code cleanup

### PR DESCRIPTION
**Pull request by bradymiller**
gacl code cleanup

### CODE REVIEW

File: library/classes/gacl/gacl.class.php [hunk 1]
```diff
@@ -42,10 +42,10 @@
...
 // to collect sql database login info and the utf8 flag
 // also collect the adodb libraries to support mysqli_mod that is needed for mysql ssl support
-require_once(dirname(__FILE__) . "/../library/sqlconf.php");
-require_once(dirname(__FILE__) . "/../vendor/adodb/adodb-php/adodb.inc.php");
-require_once(dirname(__FILE__) . "/../vendor/adodb/adodb-php/drivers/adodb-mysqli.inc.php");
-require_once(dirname(__FILE__) . "/../library/ADODB_mysqli_mod.php");
+require_once(dirname(__FILE__) . "/../../../library/sqlconf.php");
+require_once(dirname(__FILE__) . "/../../../vendor/adodb/adodb-php/adodb.inc.php");
+require_once(dirname(__FILE__) . "/../../../vendor/adodb/adodb-php/drivers/adodb-mysqli.inc.php");
+require_once(dirname(__FILE__) . "/../../../library/ADODB_mysqli_mod.php");
```
Comment (bradymiller) [inline library/classes/gacl/gacl.class.php]: TODO:
Look into why above is needed.

**Comment by tywrenn** [comment]
GACL seriously needs a makeover tbh

File: src/Gacl/Gacl.php [hunk 2]
```diff
@@ -1,62 +1,41 @@
...
 
+namespace OpenEMR\Gacl;
+
 /*
  * Path to ADODB.
  */
-
 if ( !defined('ADODB_DIR') ) {
 	define('ADODB_DIR', dirname(__FILE__).'/../vendor/adodb/adodb-php');
 }
```
Comment (bradymiller) [inline src/Gacl/Gacl.php]: TODO:
Look into why above is needed.

File: src/Gacl/Gacl.php [hunk 3]
```diff
@@ -1,62 +1,41 @@
...
 // to collect sql database login info and the utf8 flag
 // also collect the adodb libraries to support mysqli_mod that is needed for mysql ssl support
-require_once(dirname(__FILE__) . "/../library/sqlconf.php");
-require_once(dirname(__FILE__) . "/../vendor/adodb/adodb-php/adodb.inc.php");
-require_once(dirname(__FILE__) . "/../vendor/adodb/adodb-php/drivers/adodb-mysqli.inc.php");
-require_once(dirname(__FILE__) . "/../library/ADODB_mysqli_mod.php");
+require_once(dirname(__FILE__) . "/../../library/sqlconf.php");
+require_once(dirname(__FILE__) . "/../../vendor/adodb/adodb-php/adodb.inc.php");
+require_once(dirname(__FILE__) . "/../../vendor/adodb/adodb-php/drivers/adodb-mysqli.inc.php");
+require_once(dirname(__FILE__) . "/../../library/ADODB_mysqli_mod.php");
```
Comment (bradymiller) [inline src/Gacl/Gacl.php]: TODO:
Look into why above is needed.

**Comment by bradymiller** [comment]
This is testing well and plan to bring in soon (crossing fingers it will be easy merge to bootstrap PR :) )

**Pull request merged**